### PR TITLE
remove hardcoded value

### PR DESCRIPTION
**Pull request by ExtraHash**

### CODE REVIEW

File: src/js/wsui_main.js [hunk 1]
```diff
@@ -949,7 +951,7 @@ function handleAddressBook() {
 
     // disable payment id input for non standard adress
     function setAbPaymentIdState(addr) {
-        if (addr.length > 99) {
+        if (addr.length > CFG_ADDR_LENGTH) {
```
Comment (rixombea) [inline src/js/wsui_main.js]: should be `if (addr.length > config.addressLength) {` 
CFG_ADDR_LENGTH is not defined anywhere

same with other changes, use `config.*`

**Comment by ExtraHash** [comment]
ok, i've made an additional commit with the changes

File: src/js/wsui_main.js [hunk 2]
```diff
@@ -1972,9 +1972,9 @@ function handleSendTransfer() {
         if (maxsend) sendInputAmount.value = maxsend;
     });
 
-    sendInputFee.value = 0.1;
+    sendInputFee.value = CFG_MIN_SEND;
```
Comment (rixombea) [inline src/js/wsui_main.js]: this one should be `config.mininumSend`

File: src/js/wsui_main.js [hunk 3]
```diff
@@ -1972,9 +1972,9 @@ function handleSendTransfer() {
         if (maxsend) sendInputAmount.value = maxsend;
     });
 
-    sendInputFee.value = 0.1;
+    sendInputFee.value = config.addressLength;
```
Comment (rixombea) [inline src/js/wsui_main.js]: oops, this one is `minimumFee` not `addressLength`

Comment (ExtraHash) [reply src/js/wsui_main.js]: kek hold on

**Pull request merged**